package.json

```diff
@@ -9,8 +9,8 @@
     "mvntest": "npm test",
     "build": "vite build --mode production",
     "build:dev": "vite build --mode development --watch",
-    "lint": "tsc && prettier --check .",
-    "lint:fix": "prettier --write .",
+    "lint": "tsc && eslint && prettier --check .",
```

Comment:
Can you set it up so that eslint outputs checkstyle format on CI please so that the build gets failed properly?

https://github.com/jenkinsci/plugin-pom/#configuring-eslint-to-report-results-to-jenkins

Ideally have a normal report locally (unless is already reports nicely to console when you have the checkstyle format enabled too)

Reply:
going to merge to reduce conflicts needing to be resolved, but would be good if you could do this please